### ISSUE TITLE
DeepOPF: A Feasibility-Optimized Deep Neural Network Approach for AC Optimal Power Flow Problems. (arXiv:2007.01002v2 [eess.SY] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#4757]
DeepOPF: A Feasibility-Optimized Deep Neural Network Approach for AC Optimal Power Flow Problems. (arXiv:2007.01002v2 [eess.SY] UPDATED)<br>
https://ift.tt/38NVleZ<br>
<p>We develop an efficient Deep Neural Network (DNN) approach, named DeepOPF, for solving alternative current optimal power flow (AC-OPF) problems. The idea is to train a DNN model to predict a set of independent operating variables and then directly compute the remaining dependable variables by solving the AC power flow equations. Such a 2-stage approach guarantees that the power-flow balance equations are satisfied. Meanwhile, the difficulty lies in ensuring that the obtained solutions respect generations' operation limits, voltages, and branch flows. We tackle this challenge by employing a penalty approach in training the DNN. We apply a zero-order optimization technique in the training algorithm to compute the penalty gradients efficiently. We further derive a condition for tuning the size of the DNN according to the desired approximation accuracy. Simulation results of IEEE test cases show the effectiveness of the penalty approach and that DeepOPF can speed up the computing time by up to 35$\times$ as compared to a state-of-the-art solver, at the expense of $&lt;$0.1\% optimality loss.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
